executor/python_executor/executor.py

```
@@ -17,6 +17,7 @@
 
 logger = logging.getLogger(EXECUTOR_NAME)
 service_store: dict[str, Literal["launching", "running"]] = {}
+# 用于去重 oocana 重复发送的 job_id，job_id 一旦添加就不能移除
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)